### PR TITLE
fix: spacings pkg names

### PR DESCRIPTION
**Pull request by emmenko**
I accidentally introduced a bug when implementing the package.json generator #1642 

The package names are derived by their folder name, so the spacings components were missing the `spacings-` prefix in the name because the folders didn't have them.

### CODE REVIEW

File: design-system/package.json [hunk 1]
```diff
@@ -9,12 +9,7 @@
     "directory": "design-system"
   },
   "homepage": "https://uikit.commercetools.com",
-  "keywords": [
-    "javascript",
-    "design system",
-    "react",
-    "uikit"
-  ],
+  "keywords": ["javascript", "design system", "react", "uikit"],
```
Comment (adnasa) [inline design-system/package.json:12]: this again? 😄

**Review by tdeekens — APPROVED** [review]
Ah I guess it's the `spacings-*` thing. Looks good.

**Review by jonnybel — APPROVED** [review]

**Review by adnasa — APPROVED** [review]

File: design-system/package.json (hunk 1 above)
Comment (emmenko) [reply design-system/package.json:12]: It's changeset...I'll look into how they run the format, maybe we need to fix it there.

Comment (emmenko) [reply design-system/package.json:12]: I think this is the option that is causing the issue: https://github.com/atlassian/changesets/blob/f9ccfa3e9911718fcd95eff398ece64c8e9eef9b/packages/apply-release-plan/src/index.ts#L116

image

**Pull request merged**